HW3/Task18.py

```diff
@@ -0,0 +1,25 @@
+# Задача 18: Требуется найти в массиве A самый близкий по величине элемент к заданному числу X. Если таких элементов несколько,
+# вывести один любой. Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). Последняя строка содержит число X
+
+# *Пример:*
+# 5
+# 7 -2 3 5 1
+# 6 -> 7
+
+import random
+n = int(input('Введите количество элементов массива: '))
+lst = [random.randint(-10, 10) for i in range(n)]
+print(lst)
+
+x = int(input('Введите искомое число: '))
+comp = x ** 10
```

Comment:
А это для чего?

Reply:
Логика была такая - т.к. мы ищем максимально приближенное число к заданному, начинать сравнение надо с максимально удаленного от него. Честно говоря, это был рабочий вариант, который я хотела додумать и забыла поменять:)

Reply:
Можно сделать так
```suggestion
comp = float('inf')
```
Либо принять в качестве первого значения разницу между первым элементом списка и нашим введенным значением.